.github/workflows/release.yml

```diff
@@ -89,13 +104,13 @@ jobs:
     - name: download artifacts
       uses: actions/download-artifact@v2
       with:
-        name: nydus-artifacts
+        name: nydus-artifacts-${{ matrix.arch }}
         path: nydus-static
     - name: upload artifacts
       if: ${{ github.event_name == 'push' }}
       run: |
         tag=$(echo $GITHUB_REF | cut -d/ -f3-)
-        tarball="nydus-static-$tag-x86_64.tgz"
+        tarball="nydus-static-$tag-linux-${{ matrix.arch }}.tgz"
```

Comment:
It modifies the tarball URL. So we need to modify nydus-snapshotter release builder accordingly. Maybe Kata test need to be modified as well. /cc @liubin

Reply:
Yes, Kata Containers should also update its installation script.

https://github.com/kata-containers/tests/blob/125fda6881b1af747fa7a58ef340ab173c332754/integration/nydus/nydus_tests.sh#L42

But this will not break Kata's test, because Kata is using the fixed version described in versions.yaml.

After some time Kata can update its tests to use the latest nydus-snapshotter.